src/main/java/com/bmc/truesight/saas/jira/integration/adapter/JiraEntryEventAdapter.java

```diff
@@ -136,6 +155,84 @@ public JIRAEventResponse eventList(JsonNode responseIssuesNode, Template templat
         }
         response.setValidEventList(tsiValidEventList);
         response.setInvalidEventList(invalidEventList);
+        response.setInvalidEventIdsList(invalidEvents);
         return response;
     }
+
+    private String getMultipleValues(FieldItem fieldItem, JsonNode jsonNode) {
+        StringBuilder value = new StringBuilder();
+        int count = 0;
+        if (jsonNode != null && !jsonNode.isNull()) {
+            for (JsonNode node : jsonNode) {
+                try {
+                    if (count == 0) {
+                        value.append(node.get(fieldItem.getFetchKey()).asText());
+                    } else {
+                        value.append(",").append(node.get(fieldItem.getFetchKey()).asText());
+                    }
+                    count++;
+                } catch (Exception ex) {
+                }
+            }
+        }
+        return value.toString();
+    }
+
+    private String getArrayValues(JsonNode jsonNode) {
+        StringBuilder value = new StringBuilder();
+        ObjectMapper mapper = new ObjectMapper();
+        if (jsonNode != null && !jsonNode.isNull()) {
+            ObjectReader obReader = mapper.reader(new TypeReference<List<String>>() {
+            });
+            try {
+                List<String> condList = obReader.readValue(jsonNode);
+                value.append(condList);
+            } catch (IOException ex) {
+                java.util.logging.Logger.getLogger(JiraEntryEventAdapter.class.getName()).log(Level.SEVERE, null, ex);
+            }
+        }
+        return value.toString();
+    }
+
+    private String getIssueLinks(JsonNode jsonNode) {
+        StringBuilder value = new StringBuilder();
+        Set<String> treeSet = new TreeSet<>();
+        if (jsonNode != null && !jsonNode.isNull()) {
+            for (JsonNode node : jsonNode) {
+                try {
+                    treeSet.add(node.get("type").get("name").asText());
+                } catch (Exception ex) {
+                }
```

Comment:
Empty `catch` not good.. at least log whats going on... I see them in many places....